_inc/client/plans/plan-body.jsx

```diff
@@ -132,13 +161,13 @@ const PlanBody = React.createClass( {
 								<p>{ __( 'Earn income by allowing Jetpack to display high quality ads (powered by WordAds).' ) }</p>
 								{
 									this.props.isModuleActivated( 'wordads' ) ? (
-										<Button href={ 'https://wordpress.com/ads/earnings/' + this.props.siteRawUrl } className="is-primary">
+										<Button onClick={ () => this.trackPlansClick( 'view_earnings' ) } href={ 'https://wordpress.com/ads/earnings/' + this.props.siteRawUrl } className="is-primary">
```

Comment:
Really minor, but if I remember correctly, passing new functions as properties to underlying components was discouraged because it would always cause props changing and causing the component to go through the props update cycle. One way I see out of this would be to create a `this.trackEarningsClick` here, and additional methods for each condition. Not very DRY, I know.
Anyway, this being only a `Button` it's just a minor concern, not a problem. I'll leave it at your discretion.